draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1929,16 +1931,15 @@ by timer Ta. As soon as the initial states for the candidate pairs in the
 check list set have been set, a check is performed for a candidate pair
 within the first check list in the Running state, following
 the procedures in <xref target="sec-connectivity_check"/>. After that,
-
 whenever Ta fires the next check list in the Running state in the check list set
-is selected, and a check is performed for a candidate within that check list.
```

Comment:
I assume you're changing "selected" to "picked" so that it's clear this isn't referring to the "selected pair" concept? At one point, wasn't "SELECTED PAIR" put in all caps to make this distinction? Either way works for me though.

Reply:
I was told not to use capital letters. Peter tried to fix that in his PR, but he forgot a number of instances. Will be fixed before publication request.

Reply:
Would "chosen" be better word for this?

Reply:
Once we are done with this PR, we can look at that. At this point I just want to get rid of "selected" :)